e2e/schematics/application.test.ts

```diff
@@ -37,7 +52,9 @@ describe('Nrwl Workspace', () => {
     () => {
       newProject();
       newApp('myApp --directory=myDir --routing');
-      newLib('myLib --directory=myDir --routing --lazy --parentModule=apps/my-dir/my-app/src/app/app.module.ts');
+      newLib(
```

Comment:
have you changed the formatting options for this project?

Reply:
I didn't make any changes. I think this file just might have been formatted with a previous release of prettier

Reply:
Can you rebase? All the formatting issues should go away